sklearn/utils/estimator_checks.py

```diff
@@ -713,6 +713,9 @@ def _set_checking_parameters(estimator):
     if name in CROSS_DECOMPOSITION:
         estimator.set_params(n_components=1)
 
+    if name == "GenericUnivariateSelect":
+        estimator.set_params(score_func=f_classif)
+
```

Comment:
We can revert this change since `f_classif` is the default keyword argument.

Reply:
Done!